src/Model/ElementVirtual.php

```diff
@@ -22,7 +22,7 @@
  */
 class ElementVirtual extends BaseElement
 {
-    private static $icon = 'element-icon-virtual';
+    private static $icon = 'font-icon-link';
```

Comment:
Can we use 'font-icon-block-link' here instead? 🙂 This will then be aligned with the rest of the [block icons](https://silverstripe.github.io/silverstripe-admin/?selectedKind=Admin%2FIcons&selectedStory=Icon%20reference&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel).